### PR TITLE
Set select2 default theme to bootstrap4

### LINKED ISSUE #227: Select2 is not using select2-bootstrap4-theme
Select2 uses default theme which is the one for Bootstrap 5. The initSelect2 function should set something like this:
`var newObj = {minimumResultsForSearch: -1, theme: 'bootstrap4};`

### PR DESCRIPTION
**Pull request by web-cooking-factory**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Set correct default theme for select2
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     |no
| Fixed ticket?     | Fixes #227, Fixes #[36448](https://github.com/PrestaShop/PrestaShop/issues/36448)
| Sponsor company   | Evolutive
| How to test?      | Check Issue 36448

### CODE REVIEW

**Review by PululuK — APPROVED** [review]

**Review by boherm — APPROVED** [review]

**Comment by PululuK** [comment]
Thanks @web-cooking-factory

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
